redbot/core/bot.py

```diff
@@ -1051,6 +1058,25 @@ async def remove_shared_api_tokens(self, service_name: str, *token_names: str):
             for name in token_names:
                 group.pop(name, None)
 
+    async def remove_shared_api_services(self, service_names: Iterable[str]):
```

Comment:
I've just noticed that unlike `remove_shared_api_tokens()`, this method is using an iterable as a parameter. I think it would be better for consistency if this did it similarly to the aforementioned method:
```suggestion
    async def remove_shared_api_services(self, *service_names: str):
```

// I'm just posting the code suggestion for the method signature; the method usage and its docstring also need to be updated and I haven't made the code suggestion for that.

Sorry for missing this earlier :(